transit-ui/src/components/dict.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Lets not put this in `src/components`. Maybe, lets make a new folder inside `transit-ui`, maybe `data`